webapp/components/admin_console/push_settings.jsx

```diff
@@ -207,6 +207,7 @@ export default class PushSettings extends AdminSettings {
                 <DropdownSetting
                     id='pushNotificationContents'
                     values={[
+                        {value: 'generic_no_channel', text: Utils.localizeMessage('admin.email.genericNoChannelPushNotification', '"Send generic description with only sender name')},
                         {value: 'generic', text: Utils.localizeMessage('admin.email.genericPushNotification', 'Send generic description with user and channel names')},
```

Comment:
Sorry, I don't follow?

generic_no_channel = sender name
generic (was the pre-existing one) = sender and channel names